build.gradle

```diff
@@ -66,6 +66,7 @@ ext {
     commonsLang: 'org.apache.commons:commons-lang3:3.4',
     commonsCli: 'commons-cli:commons-cli:1.3',
     snakeyaml: 'org.yaml:snakeyaml:1.16',
+    commonmark: 'com.atlassian.commonmark:commonmark:0.9.0',
```

Comment:
We don't need to do anything here. We don't actually distribute the dependency; the users pull it at build time.